test/e2e/apibinding/apibinding_authorizer_test.go

```diff
@@ -170,6 +170,8 @@ func TestAPIBindingAuthorizer(t *testing.T) {
 
 func userConfig(username string, cfg *rest.Config) *rest.Config {
```

Comment:
We have a lot of this copy-pasta, might be time to add a helper to the e2e area that adds the users etc

Reply:
:+1: